aptos-move/block-executor/src/executor.rs

```diff
@@ -1461,7 +1457,7 @@ where
                 executor_arguments,
                 signature_verified_block,
                 base_view,
-                dynamic_change_set_optimizations_enabled,
+                true,
```

Comment:
Are there cases when `execute_transactions_sequential` has `false` here now?

Reply:
no but I wanted to actually fetch the feature flag to pass it here. let me add todo